src/unit/test_mutexqueue.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) Valkey Contributors
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "../mutexqueue.h"
+#include "test_help.h"
+#include <pthread.h>
+#include <time.h>
+
+/* Helper functions */
+static void add(mutexQueue *q, long value) {
+    unsigned long len = mutexQueueLength(q);
+    mutexQueueAdd(q, (void *)value);
+    TEST_EXPECT(mutexQueueLength(q) == len + 1);
+}
+
+static void pAdd(mutexQueue *q, long value) {
```

Comment:
```suggestion
static void priorityAdd(mutexQueue *q, long value) {
```
pAdd to me seems more like pointer (by contention), but realized it was priority after looking at the implementation.

Reply:
Changed `pAdd` to `priorityAdd`